test/unit/test_dats.py

```diff
@@ -108,8 +108,7 @@ def test_copy_mixed_subset_fails(self, backend, s, mdat):
         with pytest.raises(NotImplementedError):
             mdat.copy(op2.MixedDat([s, s]), subset=op2.Subset(s, []))
 
-    @pytest.mark.skipif('config.getvalue("backend")[0] not in ["cuda", "opencl"]')
-    def test_copy_works_device_to_device(self, backend, d1):
+    def test_copy_works_device_to_device(self, backend, d1, skip_cuda, skip_opencl):
```

Comment:
This is the other way round! This test should only run if the backend *is* CUDA or OpenCL.